services.py

```diff
@@ -21,7 +21,14 @@ def __init__(self, port: int, should_log: bool, frequency: float, decimation: Op
 services = {
   # service: (should_log, frequency, qlog decimation (optional))
   # note: the "EncodeIdx" packets will still be in the log
-  "sensorEvents": (True, 100., 100),
+  "sensorEventsDEPRECATED": (True, 100., 100),
```

Comment:
deprecated services don't need to be here, just need to be kept in the cereal structs

Reply:
updated